packages/ui/src/views/pull-request/pull-request-list-page.tsx

```diff
@@ -0,0 +1,119 @@
+import { Link } from 'react-router-dom'
+
+import { NoData } from '@components/no-data'
+import { noop } from 'lodash-es'
+
+import { SandboxLayout } from '..'
+import { Button, PaginationComponent, Spacer, Text } from '../../components/index'
+import { PullRequestList } from './pull-request-list'
+import { PullRequestStore, PullRequestType, RepoRepositoryOutput } from './types'
+
+export interface PullRequestListProps {
+  usePullRequestStore: () => PullRequestStore
+  repoId?: string
+  spaceId?: string
+  repoMetadata?: RepoRepositoryOutput
+}
+
+const PullRequestListPage: React.FC<PullRequestListProps> = ({
```

Comment:
Let's not use `Page` in the name of any of the views. The `Page` will be in the consuming app.
```suggestion
const PullRequestList: React.FC<PullRequestListProps> = ({
```

Reply:
done